stock_prices/stock_prices.py

```diff
@@ -1,15 +1,31 @@
 #!/usr/bin/python
 
-import argparse
+# import argparse
 
 def find_max_profit(prices):
-  pass
+  profitArr = []
+  highest = prices[1]-prices[0]
+  for i in range(0, len(prices)):
+    for j in range(0, len(prices)): 
+      if i < j:
+        profit = prices[j] - prices[i]
+        profitArr.append(profit)
+  j+=1     
+  for k in range(0, len(profitArr)):
```

Comment:
Do we need a profitArr?  or could we just have some single value variable for something like `highest_profit` that is updated when necessary instead?

Either way, this does not increase past linear complexity so it is a good working solution.

Good job with these 2 problems and your assigned whiteboarding problem.